CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateTournament.cpp

```diff
@@ -45,7 +49,7 @@ void Kartaclysm::StateTournament::Enter(const std::map<std::string, std::string>
 	HeatStroke::EventManager::Instance()->AddListener("RaceInfo", m_pRaceInfoDelegate);
 
 	// Shuffle tracks for tournament and push to player selection
-	std::random_shuffle(m_vTracks.begin(), m_vTracks.end());
```

Comment:
I'm assuming the reason this wasn't working is because you didn't seed it, and the value of this function when given the default seed just happens to be the correct order (we only have three tracks, so it's not unreasonable).  Regardless, switching to the new method is likely the best option, especially considering std::random_shuffle is deprecated in C++14.